Assets/js/main.js

```diff
@@ -3,13 +3,11 @@ import bannerCarouselComponent from './banner-carousel.js';
 import bannerVideoComponent from './banner-video.js';
 import newFn from './new.js';
 import headerFn from './header.js';
-import successStoriesFn from './success-stories-banner.js';
 
 const app = new App();
 
 const bannerCarouselSlider = bannerCarouselComponent();
 
 const plasticBannerVideoComp = bannerVideoComponent();
 const newFnComp = newFn();
-const headerComponent = headerFn();
-const successStoriesComponent = successStoriesFn();
+const headerComponent = headerFn();
```

Comment:
@Satyam1RP You've removed success stories reference from this file, any reason for that.
Cc:-  @jvaishnavrp

Reply:
@Lalit-gour1 This happened unintentionally at the time of file selection, I fixed and pushed this code.
CC: @jvaishnavrp